cmd/umoci/insert.go

```diff
@@ -0,0 +1,140 @@
+/*
+ * umoci: Umoci Modifies Open Containers' Images
+ * Copyright (C) 2018 Cisco
```

Comment:
A lot of this file is copied from the other `cmd/umoci/insert.go` files -- can you add an additional copyright line from the original?

Reply:
Sure, np